README.markdown

```diff
@@ -260,6 +260,15 @@ class MyViewController: UIViewController, UITableViewDataSource, UIScrollViewDel
 }
 ```
 
+When putting protocol conformance for an object in an extension as previously mentioned, consider creating a folder and separated files to put its ancillary extensions therefore reducing the file of the initial component and facilitating looking for specific conformance.
+
+The container folder should have the name of the component (E.g., MyViewController) and inside the folder the main component should be placed alongside with the extensions, which will be named using the following convention: `ComponentName+ConformanceName`.
+
+**Preferred:**
+
+![Protocol Conformance Extensions](screens/protocolConformanceExtensions.png)
+
```

Comment:
@NSLeoFont I'll also add an ASCII art folder structure, or better in place of a GIF. Leaner and cleaner. Here you have it:

```
[MyViewController] (folder)
|- MyViewController.swift
|- MyViewController+SomeDelegate.swift
|- MyViewController+SomeViewProtocol.swift
|- MyViewController+UITableViewDelegate.swift
|- MyViewController+Themable.swift
|- MyViewController+StoryboardInstantiable.swift
|- MyViewController+UITextViewDelegate.swift


```

Reply:
Long life to the 80s! Viva ascii-art! Done in 7250d35473cbe05098ee6c909d0c687bba966e3f